src/Components/calculator.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './calculator.css';
+
+const Button = () => {
+  const numbers = ['AC', '+/-', '%', '÷', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
+  return (
+    <div>
+      <form action="">
+        <input type="text" value="No result" className="input-result" />
+      </form>
+
+      <ul className="numbers-container">
+        {numbers.map((number) => (
+          <li className="number" key={number}>{number}</li>
+        ))}
+      </ul>
+    </div>
+  );
+};
```

Comment:
- [ **OPTIONAL** ] For React best practices, I think it would be better if you put the above code lines in separate components files then import in the Calculator component file. Or you can divide your code in the way you want but ensure that the two components are not in the same file. Good job with your design.